mailbag/formats/eml.py

```diff
@@ -22,41 +18,49 @@ class EML(EmailAccount):
     """EML - This concrete class parses eml file format"""
     format_name = 'eml'
 
-    def __init__(self, target_account, **kwargs):
+    def __init__(self, target_account, args, **kwargs):
         log.debug("Parsity parse")
         # code goes here to set up mailbox and pull out any relevant account_data
         account_data = {}
 
         self.file = target_account
-        #self.dry_run = dry_run
-        #self.mailbag_name = mailbag_name
+        # self.dry_run = dry_run
+        # self.mailbag_name = mailbag_name
         log.info("Reading : ", File=self.file)
 
     def account_data(self):
         return account_data
 
-
     def messages(self):
 
         files = glob.glob(os.path.join(self.file, "**", "*.eml"), recursive=True)
 
         for i in files:
-            #log.debug(i)
+            attachmentNames = []
+            attachments = []
             with open(i, 'r') as f:
-                #a = f.read()
-                msg = email.message_from_file(f)
-
+                msg = email.message_from_file(f, policy=policy.default)
+                body = msg.get_body(preferencelist=('related', 'html', 'plain')).__str__()
 
+                # Extract Attachments                
+                for attachment in msg.iter_attachments():
```

Comment:
`iter_attachments` is silently not detecting the attachment for the WIRELESS CALLER message. For now I'll just write it up as an issue.